### PR TITLE
[WW-5338] Removes deprecated OgnlTool

### PR DESCRIPTION
**Pull request by lukaszlenart**
Closes [WW-5338](https://issues.apache.org/jira/browse/WW-5338)

### CODE REVIEW

File: core/src/main/java/org/apache/struts2/util/StrutsUtil.java [hunk 1]
```diff
@@ -125,12 +120,10 @@ public Object findValue(String expression, String className) throws ClassNotFoun
 
     public Object findValue(String expr, Object context) {
         try {
-            return ognl.getValue(expr, ActionContext.getContext().getContextMap(), context);
-        } catch (OgnlException e) {
-            if (e.getReason() instanceof SecurityException) {
-                LOG.error(format("Could not evaluate this expression due to security constraints: [{0}]", expr), e);
-            }
-            return null;
+            stack.push(context);
```
Comment (kusalk) [inline core/src/main/java/org/apache/struts2/util/StrutsUtil.java:123]: Should be outside the try block so that we don't pop unless successfully pushed

**Review by kusalk — COMMENTED** [review]
We can also delete `org.apache.struts2.views.util.ContextUtil#OGNL`

File: core/src/main/java/org/apache/struts2/util/StrutsUtil.java (hunk 1 above)
Comment (lukaszlenart) [reply core/src/main/java/org/apache/struts2/util/StrutsUtil.java]: fixed

File: plugins/velocity/src/main/java/org/apache/struts2/views/velocity/VelocityManager.java [hunk 2]
```diff
@@ -56,8 +56,7 @@
 import static java.lang.String.format;
 import static java.util.Collections.emptyList;
 import static java.util.stream.Collectors.joining;
-import static java.util.stream.Collectors.toList;
-import static org.apache.struts2.views.util.ContextUtil.OGNL;
+import static java.util.stream.Collectors.toList;;
```
Comment (kusalk) [inline plugins/velocity/src/main/java/org/apache/struts2/views/velocity/VelocityManager.java:59]: Double semicolon 😁

**Review by kusalk — APPROVED** [review]
LGTM

File: plugins/velocity/src/main/java/org/apache/struts2/views/velocity/VelocityManager.java (hunk 2 above)
Comment (lukaszlenart) [reply plugins/velocity/src/main/java/org/apache/struts2/views/velocity/VelocityManager.java]: 😵‍💫

**Pull request merged**